sdk/nodejs/types/input.ts

```diff
@@ -7527,7 +7638,7 @@ export namespace cloudrun {
         /**
          * Number of seconds after which the probe times out.
          * Defaults to 1 second. Minimum value is 1. Maximum value is 3600.
-         * Must be smaller than periodSeconds.
+         * Must be smaller than period_seconds.
```

Comment:
Doesn't look right (albeit, it's a doc change). Where is `period_seconds` anyway?

Reply:
Yeah, that looks like another conversion error. I see `periodSeconds` in `ServiceTemplateSpecContainerLivenessProbe`.